content/blog/2019/07/2019-07-24-azure-artifact-manager.adoc

```diff
@@ -0,0 +1,107 @@
+Here’s are a few of commonly used artifacts related steps in pipeline jobs; all are supported to push artifacts to the Azure Storage blob specified.
+
+You can use _archiveArtifacts_ step to archive target artifacts into Azure storage. For more details about _archiveArtifacts_ step, see the Jenkins https://jenkins.io/doc/pipeline/steps/core/#archiveartifacts-archive-the-artifacts[archiveArtifacts setp documentation].
+
+----
+node {
+     //...
+     stage('Archive') { 
+        archiveArtifacts "pattern"
+     } 
+}
+----
+
+You can use the _unarchive_ step to retrieve the artifacts from Azure storage. For more details about _unarchive_ step, please see https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-unarchive-code-copy-archived-artifacts-into-the-workspace[unarchive step documentation].
+
+----
+node {
+     //...
+     stage('Unarchive') {
+        unarchive mapping: ["pattern": '.']
+     } 
+}
+----
+
+To save a set of files so that you can use them later in the same build (generally on another node or workspace), you can use _stash_ step to store files into Azure storage for later use. _Stash_ step documentation can be found https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#unstash-restore-files-previously-stashed[here].
+
+----
+node {
+     //...
+     stash name: 'name', includes: '*'
+}
+----
+
+You can use _unstash_ step to retrieve the files saved with _stash_ step from Azure storage to the local workspace.  _Unstash_ documentation can be found https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#unstash-restore-files-previously-stashed[here].
+
+----
+node {
+     //...
+     unstash 'name'
+}
+----
+
+====== FreeStyle Job
+
+For a FreeStyle Jenkins job, you can use *Archive the artifacts* step in *Post-build Actions* to upload the target artifacts into Azure storage.
+
+image:/images/post-images/2019-07-24-azure-artifact-manager/4.post-build-actions.png[role=center]
+
+This Azure Artifact Manager plugin is also compatible with some other popular management plugins, such as the https://plugins.jenkins.io/copyartifact[Copy Artifact] plugin. You can still use these plugins without changing anything.
+
+image:/images/post-images/2019-07-24-azure-artifact-manager/5-build.png[role=center]
+
+== Troubleshooting
+
+If you have any problems or suggestions when using Azure Artifact Manager plugin, you can file a ticket on https://issues.jenkins-ci.org/secure/Dashboard.jspa[Jenkins JIRA] for the azure-artifact-manager-plugin component.
+
+== Conclusion
+
+In summary, the https://plugins.jenkins.io/azure-artifact-manager[Azure Artifact Manager] enables a more cloud-native Jenkins. This is the first step in the https://jenkins.io/sigs/cloud-native/[Cloud Native project]. We have a long way to go to get Jenkins to run on cloud environments as a true “Cloud Native” application. We need help and welcome your participation and contributions to make Jenkins better. Please start contributing and/or give us feedback!
```

Comment:
I don't think the in summary is needed?

```suggestion
The https://plugins.jenkins.io/azure-artifact-manager[Azure Artifact Manager] enables a more cloud-native Jenkins. This is the first step in the https://jenkins.io/sigs/cloud-native/[Cloud Native project]. We have a long way to go to get Jenkins to run on cloud environments as a true “Cloud Native” application. We need help and welcome your participation and contributions to make Jenkins better. Please start contributing and/or give us feedback!
```

Reply:
You are not wrong, but I am not the author of the post. I'm happy respecting the author's text. 

More concerned over the errors that are being generated and the build failure :)